site/pom.xml

```diff
@@ -21,7 +21,7 @@
     <!-- P R O J E C T  D E S C R I P T I O N                                   -->
     <!-- ====================================================================== -->
     <groupId>com.adobe.aem.guides</groupId>
-    <artifactId>aem-sites-template-basic.site.template</artifactId>
+    <artifactId>aem-sites-template-basic.site</artifactId>
     <version>0.0.1-SNAPSHOT</version>
     <packaging>content-package</packaging>
     <name>WKND Sites StarterKit - Site Template</name>
```

Comment:
Could we get that renamed to `AEM Sites Basic Template` ?

Reply:
> Why did you remove `.env` file from old `site.theme` folder instead of moving it to `site` folder?

The file is still there: under the theme folder
site.theme -> theme

Reply:
I'd go with `AEM Basic Site Template`